src/dash/vo/DescriptorType.js

```diff
@@ -45,6 +45,7 @@ class DescriptorType {
             this.value = data.value ? data.value : null;
             this.id = data.id ? data.id : null;
         }
+        return this;
```

Comment:
I removed this on purpose, can we use the same logic like we did for supplemental properties:

````
 s.init(supp);
return s
````